frontend/src/components/ViewActivities/activitylist.js

```diff
@@ -8,17 +8,33 @@ import app from '../Firebase';
 const db = app.firestore();
 
 /**
- * Gets the list of activities from the server. 
+ * ReactJS class component for the list of activities. 
  * 
- * @param {string} tripId The trip ID.
- * @return {ActivityInfo[]} The list of trip activities.
+ * @property {Object} props ReactJS props.
+ * @property {string} tripId The tripID.
```

Comment:
Consistency with JSDoc for props in `activity.js` (choose one way or other)
```suggestion
 * @property {string} props.tripId The tripID.
```